entries/f/fragdenstaat.de.json

```diff
@@ -0,0 +1,13 @@
+{
+  "FragDenStaat": {
+    "domain": "fragdenstaat.de",
+    "tfa": [
+      "totp",
+      "u2f"
```

Comment:
Is this implemented as a passwordless authentication or a second factor?

Reply:
afaik 2nd factor "only"